aries-site/src/data/wcag/fileinput.json

```diff
@@ -170,8 +170,8 @@
   {
     "rule": "1.4.13",
     "label": "Content on Hover or Focus",
-    "status": "failed",
-    "notes": "Failing because when the file name truncates there isn't a way for keyboard users to access the tooltip"
+    "status": "passed",
+    "notes": "Initial note - Failing because when the file name truncates there isn't a way for keyboard users to access the tooltip. We might need to revisit this in the future. Updated note - At the moment we are recommending that implementers use `renderFile` prop to display full name in cases where they need to display the full file name."
```

Comment:
```suggestion
    "notes": "Initial note - Failing because when the file name truncates there isn't a way for keyboard users to access the tooltip. We might need to revisit this in the future. Updated note - At the moment, we are recommending that implementers use the renderFile prop to display the full file name in cases where truncation occurs and the full name needs to be visible. The default tooltip uses the native HTML title attribute, which is not accessible to keyboard users. Since this is a browser-level limitation and not an issue with Grommet itself, we're unable to address it directly."
```